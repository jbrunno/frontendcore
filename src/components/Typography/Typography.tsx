import React, { ElementType } from 'react';
import {
  Typography as MuiTypography,
  TypographyProps as MuiTypographyProps,
} from '@mui/material';

export type TypographyProps = MuiTypographyProps & {
  component?: ElementType;
};

export function Typography(props: TypographyProps) {
  return <MuiTypography {...props} />;
}

export default Typography;
