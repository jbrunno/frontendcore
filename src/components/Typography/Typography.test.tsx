import React from 'react';
import { render } from '@test-utils/component-testing';
import Typography from './Typography';

test('renders the Typography', () => {
  const { container } = render(<Typography />);
  expect(container).toMatchSnapshot();
});
