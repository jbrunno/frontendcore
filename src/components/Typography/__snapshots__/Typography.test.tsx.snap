// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Typography 1`] = `
.emotion-0 {
  margin: 0;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5rem;
  letter-spacing: 0.0094rem;
  font-family: Lato,sans-serif;
}

<div>
  <p
    class="MuiTypography-root MuiTypography-body1 emotion-0"
  />
</div>
`;
